### ISSUE TITLE
Fly bug or Fly hack

### ISSUE THREAD

**Issue by Blushifix** [WarEmu/WarBugs#16290]
**Expected behavior and actual behavior:**
Player (Snuskigt) was able to fly / hover above he's scenario group while casting healing spells for a really long time. Either he hit an area where he could not recover from falling or might have been abusing a bug.
The time when issue occurred: 22:05 CET 

Players should not be able to hover over their scenario group out of reach of other players.

**Steps to reproduce the problem:**
Unfortunately, I only saw him jump at a tree just before entering the middle flag capture from destruction side.

**Testing Screenshots/Videos/Evidences (always needed):**

https://imgur.com/a/6uUFGx1
https://i.gyazo.com/b3ea2c32f4b5a71a6fc730349065c951.mp4

**Comment by Blushifix** [comment]
I included a screenshot of a scenario today where the same thing happened. Might be a visual bug that gets triggered if you get knocked up near the tree in Nordenwatch. However not sure.

**Comment by Kaelang** [comment]
Are you only seeing this happen in Nordenwatch?

**Comment by Blushifix** [comment]
Yes only had this issue when someone gets knocked up I think in Nordenwatch near the tree at the middle flag point destruction side.
